app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const angular = require('angular');
+
+import { PipelineTemplates, StageConstants } from '@spinnaker/core';
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.enableAsgStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'enableServerGroup',
+      alias: 'enableAsg',
+      cloudProvider: 'ecs',
+      templateUrl: require('./enableAsgStage.html'),
+      executionDetailsUrl: PipelineTemplates.enableAsgExecutionDetails,
```

Comment:
PipelineTemplates.enableAsgExecutionDetails no longer exists, you should be able to just omit the executionDetailsUrl property and the default execution details sections will be picked up from the default destroy asg stage.

Reply:
Done